python/ml4ir/base/model/architectures/dnn.py

```diff
@@ -122,6 +122,7 @@ def call(self, inputs, training=None):
             Logits tensor computed with the forward pass of the architecture layer
         """
         train_features = inputs[FeatureTypeKey.TRAIN]
+        metadata_features = inputs[FeatureTypeKey.METADATA]
```

Comment:
This was a bug in the previous update. Let me know if this looks right @lastmansleeping